### PR TITLE
Small quality of life fixups

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: src/BenchmarksClient/Workers/SignalRWorker.cs [hunk 1]
```diff
@@ -311,9 +319,16 @@ private void CalculateStatistics()
             Log($"Least Requests per Connection: {min}");
             Log($"Most Requests per Connection: {max}");
 
+            if (_workTimer.ElapsedMilliseconds <= 0)
```
Comment (sebastienros) [inline src/BenchmarksClient/Workers/SignalRWorker.cs]: What does it do?

Comment (BrennanConroy) [reply src/BenchmarksClient/Workers/SignalRWorker.cs]: Prevents the `NaN` and thus the failed SQL query

File: src/BenchmarksClient/Workers/SignalRWorker.cs [hunk 2]
```diff
@@ -60,8 +60,10 @@ public SignalRWorker(ClientJob job)
                 jobLogText += $" TransportType:{transportType}";
             }
 
-            jobLogText += "]";
-            JobLogText = jobLogText;
+            if (_job.ClientProperties.TryGetValue("HubProtocol", out var protocol))
+            {
+                jobLogText += $" HubProtocol:{protocol}";
```
Comment (davidfowl) [inline src/BenchmarksClient/Workers/SignalRWorker.cs]: 😄

**Comment by BrennanConroy** [comment]
Ping

**Pull request merged**